source/Calamari/Kubernetes/Integration/Kubelogin.cs

```diff
@@ -24,13 +24,17 @@ public bool TrySetKubeLogin()
             if (string.IsNullOrEmpty(foundExecutable))
             {
                 log.Error("Could not find kubelogin. Make sure kubelogin is on the PATH.");
```

Comment:
I think this should be updated to something along the lines of `log.Verbose("Could not find  kubelogin on PATH. The kubeconfig will not be converted to use the Exec Plugin.");`